src/output-json-stats.c

```diff
@@ -395,7 +395,7 @@ static OutputInitResult OutputStatsLogInitSub(ConfNode *conf, OutputCtx *parent_
         return result;
     }
 
-    OutputStatsCtx *stats_ctx = SCMalloc(sizeof(OutputStatsCtx));
+    OutputStatsCtx *stats_ctx = SCCalloc(1, sizeof(OutputStatsCtx));
```

Comment:
There is no memset here, so the commit message should be changed, right ?

Reply:
And given that every field is set after the malloc, I do not think this commit is needed, right ?

Reply:
I'll remove this commit.